.github/workflows/release.yml

```diff
@@ -31,6 +31,9 @@ jobs:
           # TODO: Replace with go-version-from-file when it is supported
           # https://github.com/actions/setup-go/pull/62
           go-version: ${{ steps.go-version.outputs.content }}
+      -
+        name: Generate provider schemas
+        run: go generate ./internal/schemas
```

Comment:
This makes me realize that users installing LS via `go install/get` will be missing these schemas.
This would likely also affect the community-maintained Homebrew formula https://formulae.brew.sh/formula/terraform-ls

Even if we (re)introduce `make build` target, which was removed as part of https://github.com/hashicorp/terraform-ls/pull/98 the remaining problem is that we have no way of communicating there is any makefile and that it may need to run. This was one of the reasons I initially actively prevented `go get` installations as per https://github.com/hashicorp/terraform-ls/issues/88#issuecomment-625780914 Also I would rather avoid adding a makefile target which isn't being actively used/tested in our CI.

I think the best way forward would be to keep encouraging people to just use our builds and make sure we can more easily identify unofficial builds as per https://github.com/hashicorp/terraform-ls/issues/127 if people experience bugs in those. Then builders of unofficial builds will either keep following changes in our CI configs and add `go generate` on their side, or produce builds with missing schema, which may be just fine? 🤷🏻‍♂️ 

We started baking the schemas in based on the feedback from VS Code extension users, all/most of which have _our_ builds of LS installed automatically - so it shouldn't affect that critical majority.

--- 

Longer term this may not be an issue though once we stop compiling schemas into the binary and start pulling them from the Registry.

--- 

All in all I would just add a note about this into the Changelog once merging it, to give folks a chance to spot it and adapt their build configs.

Reply:
cc @chenrui333 ^ (I noticed you contributed some recent updates to [the Homebrew formula](https://github.com/Homebrew/homebrew-core/commits/b3a83e2fdec689b58a337a1b29eb0eb02b2aa1db/Formula/terraform-ls.rb))